pytorch_lightning/accelerators/accelerator.py

```diff
@@ -287,10 +287,15 @@ def optimizer_zero_grad(self, current_epoch: int, batch_idx: int, optimizer: Opt
         model_ref = self.lightning_module
         model_ref.optimizer_zero_grad(current_epoch, batch_idx, optimizer, opt_idx)
 
-    def clip_gradients(self, optimizer: Optimizer, clip_val: Union[int, float]) -> None:
+    def clip_gradients(
+            self,
```

Comment:
mind format with yapf, run in terminal `yapf . -rip`

Reply:
updated.